heft-plugins/heft-jest-plugin/heft-plugin.json

```diff
@@ -36,6 +36,11 @@
           "argumentName": "SOURCE_FILE",
           "description": "Find and run the tests that cover a source file that was passed in as an argument.  This corresponds to the \"--findRelatedTests\" parameter in Jest's documentation. This parameter is not compatible with watch mode."
         },
+        {
+          "longName": "--log-heap-usage",
+          "parameterKind": "flag",
+          "description": "Logs the heap usage after every test. Useful to debug memory leaks. Use together with --expose-gc in node."
```

Comment:
Is this functionality useful if the `--expose-gc` parameter isn't provided? If it isn't, we should ensure it was passed and print a warning if it wasn't.

Reply:
I tested it without --expose-gc and it worked perfectly. I'm not really sure why they say this in Jest documentation when it's not absolutley neceseery. Mybe remove the mention of it for the time being.

Reply:
Does that depend on the version of Node?